pkg/triggers/triggers.go

```diff
@@ -58,6 +58,13 @@ func ExposeEventListner(c *clients.Clients, elname, namespace string) string {
 	return GetRoute(elname, namespace)
 }
 
+func ExposeDeploymentConfig(c *clients.Clients, elname, namespace string) string {
```

Comment:
Port could also be a parameter so that the function is reusable.

Reply:
Done